packages/big-design/src/components/FileUploader/DropZone.tsx

```diff
@@ -191,15 +222,19 @@ export const DropZone = ({
               {renderedDescription}
             </Flex>
           </Flex>
-          <StyledButton
-            color="secondary"
-            disabled={disabled}
-            marginTop={isRowView ? 'none' : 'medium'}
-            onClick={(e) => e.preventDefault()}
-            variant="subtle"
-          >
-            {localization.upload}
-          </StyledButton>
+          <Flex>
```

Comment:
🍹 Might want to enforce the flex direction for mobile devices
```suggestion
          <Flex flexDirection="row">
```

Reply:
Done